ibft/ibft.go

```diff
@@ -148,7 +152,17 @@ func (i *ibftImpl) GetIBFTCommittee() map[uint64]*proto.Node {
 	return i.ValidatorShare.Committee
 }
 
+// GetIdentifier returns ibft identifier made of public key and role (type)
+func (i *ibftImpl) GetIdentifier() []byte {
+	return i.Identifier
+}
+
 // resetLeaderSelection resets leader selection with seed and round 1
 func (i *ibftImpl) resetLeaderSelection(seed []byte) error {
 	return i.leaderSelector.SetSeed(seed, 1)
 }
+
+// IdentifierFormat return base format for lambda
+func IdentifierFormat(pubKey []byte, role beacon.Role) string {
```

Comment:
@nivBlox are we using it only in the ibft implementation? 
Feels like this should be in some util function